distutils/tests/test_filelist.py

```diff
@@ -45,22 +46,22 @@ def assertWarnings(self, caplog):
         caplog.clear()
 
     def test_glob_to_re(self):
-        sep = os.sep
-        if os.sep == '\\':
-            sep = re.escape(os.sep)
+        sep = re.escape(os.sep) if os.sep == '\\' else os.sep
```

Comment:
Alternatively:
```suggestion
        sep = r"\\" if os.sep == "\\" else os.sep
```

or even:
```suggestion
        sep = re.escape(os.sep)
```